pkg/agent/sysadvisor/plugin/qosaware/resource/memory/advisor.go

```diff
@@ -46,6 +46,7 @@ func init() {
 	memadvisorplugin.RegisterInitializer(memadvisorplugin.CacheReaper, memadvisorplugin.NewCacheReaper)
 	memadvisorplugin.RegisterInitializer(memadvisorplugin.MemoryGuard, memadvisorplugin.NewMemoryGuard)
 	memadvisorplugin.RegisterInitializer(memadvisorplugin.MemsetBinder, memadvisorplugin.NewMemsetBinder)
+	memadvisorplugin.RegisterInitializer(memadvisorplugin.MemoryProvisioner, memadvisorplugin.NewMemoryProvisioner)
```

Comment:
why this `memory provisioner` be in the level of `plugin` instead of in the level of `headroom policy` (provision policy, for instance)

Reply:
In the previous discussion, we want to reuse the `ListAndWatchResponse` of `memoryServer` to interact with `qrm mremory plugin`, which means that `memory provisioner` needs to be a Plugin.